src/input.cpp

```diff
@@ -419,6 +419,16 @@ const std::vector<input_event> &input_manager::get_input_for_action( const std::
     return attributes.input_events;
 }
 
+char input_manager::get_first_char_for_action( const std::string &action_descriptor,
+        const std::string context )
+{
+    const input_event first_input_event = get_input_for_action( action_descriptor, context )[0];
```

Comment:
How can you be sure that the vector returned by `get_input_for_action` always has at least one entry?

Reply:
I've added vector size check.